Add billing note (#17666)
DataDog/documentation: content/en/real_user_monitoring/connect_rum_and_traces/_index.md

### Setup RUM

> **Note:** Configuring RUM and Traces makes use of RUM paid data in APM, which may impact your billing.

Author: ```suggestion
**Note:** Configuring APM spans to connect to RUM data makes use of RUM paid data in APM, which may impact your billing.
```

Author: @jamiemilstein I have a few thoughts/questions:
- This note actually does route to the APM docs page
- Does the update above make it any better/clearer? I'm open to any rephrasing suggestions you have
- I just realized something - would it would be better to move the note to the last paragraph under the **Overview** section?

Reviewer: yes i think it's better! i think the only thing I would swap is _APM paid data in RUM_ as we are talking about viewing traces in RUM. 
@hugokash does this make sense from your side of things? since we're talking about billing wanted to get a second set of eyes

Reviewer 2: That's ok. Should we specify "may impact your APM billing" ?

Author: @hugokash sure! Thanks for your input as well @jamiemilstein.

> 

Reviewer: i think it's the opposite actually -- if you configure APM spans to connect to RUM data it impacts your billing.

my thought process is if we did this on the APM docs page the statement you added is accurate but in this case i think it's the other way around, wdyt?

> **Note:** Configuring APM spans to connect to RUM data makes use of APM paid data in RUM, which may impact your APM billing.

Author: ```suggestion
**Note:** Configuring RUM and Traces makes use of APM paid data in RUM, which may impact your APM billing.
```
